src/arduino_lmic_hal_configuration.h

```diff
@@ -97,9 +97,9 @@ class HalConfiguration_t
 	// PA_BOOST mode. So that's our default. Override this
 	// for the Murata module.
 	virtual TxPowerPolicy_t getTxPowerPolicy(
-		TxPowerPolicy_t policy,
-		int8_t requestedPower,
-		uint32_t frequency
+		__attribute__((unused)) TxPowerPolicy_t policy,
```

Comment:
`LMIC_UNREFERENCED_PARAMETER()` is for parameters that are not used (and could be removed if we weren't lazy or conservative).

`LMIC_API_PARAMETER()` is for parameters that are part of the API definition, but not used in a given implementation of the API.

There are a number of others for other situations. In this case, you want to use LMIC_API_PARAMETER(). These are used in the body of the function where a statement is legal.  There are example if you search. Where possible, the LMIC code tries to hide GCC-isms in macros. This doesn't always get updated, depends on porting history of the code to other environments.

Reply:
Thanks, done in next commit